### PR TITLE
Add duration (by default) to tblock command

### PR DESCRIPTION
**Pull request by Lpsd**
Updates the `.tblock` command, adding an extra argument for block duration. The argument is optional and defaults to "5d" (5 days).

Usage: `.tblock <userid> <duration>`

Duration is a string e.g `1 day`, `1d`, `2 months`, `60s`

Adds a command to check if user is blocked, `.tcheckblock <userid>`

### CODE REVIEW

**Review by xLive — APPROVED** [review]
I would use [Discord timestamp format](https://gist.github.com/LeviSnoot/d9147767abeef2f770e9ddcd91eb85aa) instead of `fromtimestamp`, but other than that, LGTM :shipit:

File: forward/forward.py [hunk 1]
```diff
@@ -240,28 +271,43 @@ async def tpm(self, ctx, language, user: discord.Member, *, message: str):
...
         """
         async with self.config.blocked() as blocked:
             userid = str(user.id)
             if userid not in blocked:
-                blocked[userid] = True
-                await ctx.maybe_send_embed("Blocked <@{}> from send messages to the bot.".format(userid))
+                duration = str_to_timedelta(duration=time).get("duration")
+                duration = duration if type(duration) is timedelta else str_to_timedelta(duration="5d").get("duration")
+                blocked[userid] = (datetime.now(timezone.utc) + duration).timestamp() # until
+                await ctx.maybe_send_embed("Blocked <@{id}> from sending messages to the bot until {until}.".format(id=userid, until=datetime.fromtimestamp(blocked[userid])))
```
Comment (theSarrum) [inline forward/forward.py:283]:
```suggestion
                await ctx.maybe_send_embed("Blocked <@{id}> from sending messages to the bot until <t:{until}>.".format(id=userid, until=int(blocked[userid])))
```

File: forward/forward.py [hunk 2]
```diff
@@ -240,28 +271,43 @@ async def tpm(self, ctx, language, user: discord.Member, *, message: str):
...
             else:
-                await ctx.maybe_send_embed("This user is already blocked.")
+                await ctx.maybe_send_embed("This user is already blocked until {}.".format(datetime.fromtimestamp(blocked[userid])))
```
Comment (theSarrum) [inline forward/forward.py:285]:
```suggestion
                await ctx.maybe_send_embed("This user is already blocked until <t:{}>.".format(int(blocked[userid])))
```

File: forward/forward.py [hunk 3]
```diff
@@ -240,28 +271,43 @@ async def tpm(self, ctx, language, user: discord.Member, *, message: str):
...
+    @commands.command()
+    @commands.guild_only()
+    @checks.guildowner()
+    async def tcheckblock(self, ctx, user: discord.Member):
+        """Checks if a member is blocked from sending dms to the bot
+        """
+        async with self.config.blocked() as blocked:
+            userid = str(user.id)
+            if userid in blocked:
+                await ctx.maybe_send_embed("User <@{id}> is blocked until {until}.".format(id=userid, until=datetime.fromtimestamp(blocked[userid])))
```
Comment (theSarrum) [inline forward/forward.py:310]:
```suggestion
                await ctx.maybe_send_embed("User <@{id}> is blocked until <t:{until}>.".format(id=userid, until=int(blocked[userid])))
```

**Review by theSarrum — CHANGES_REQUESTED** [review]
Let's make the changes that @xLive suggested.

**Pull request merged**